galleries/examples/color/color_sequences.py

```diff
@@ -0,0 +1,66 @@
+"""
+=====================
+Named Color Sequences
+=====================
+
+Matplotlib's `~matplotlib.colors.ColorSequenceRegistry` allows access to
+predefined lists of colors by name e.g.
+``colors = matplotlib.color_sequences['Set1']``.  This example shows all of the
+built in color sequences.
+
+Note that user-defined sequences can be added via the registry's
+`~matplotlib.colors.ColorSequenceRegistry.register` method.
```

Comment:
```suggestion
`.ColorSequenceRegistry.register`.
```

Reply:
This renders how I actually wanted it but wasn't sure if it was possible!